.dagger/build/sdk.go

```diff
@@ -76,6 +76,7 @@ const TypescriptSDKTSXVersion = "4.15.6"
 func (build *Builder) typescriptSDKContent(ctx context.Context) (*sdkContent, error) {
 	tsxNodeModule := dag.Container(dagger.ContainerOpts{Platform: build.platform}).
 		From(tsdistconsts.DefaultNodeImageRef).
+		WithExec([]string{"npm", "install", "-g", "npm"}).
```

Comment:
:eyes: is this connected?

Reply:
That line is quite confusing yeah 👀

Reply:
Oh sorry, I found the build stuck during install tsx on my machine and that issue is resolved when upgrading the npm.

But yeah, it is not related to this PR and should get remove.

Reply:
```suggestion
```